src/site/apt/examples/custom-property-expansion.apt.vm

```diff
@@ -68,7 +68,10 @@ Using Custom Checkstyle Property Expansion Definitions
         <version>${project.version}</version>
         <configuration>
           <configLocation>checkstyle.xml</configLocation>
-          <propertyExpansion>projectname=whizbang</propertyExpansion>
+          <propertyExpansion>
+            projectname=whizbang
+            anotherProperty=bazinga
+          </propertyExpansion>
```

Comment:
oh, this parameter should have `Map` type instead `String` ... but it is another story 😄

Reply:
ah! Ya, I didn't dig in to look at the code (tho I'd like to when I have the time), I just figured this out locally and figured I'd leave it as a breakcrumb for other people with the same question :)

Reply:
we can deprecate `propertyExpansion` and add new 

```
Map<String, String> propertyExpansions;
```

then we can use in configuration:

```
<propertyExpansions>
 <key1>value</key1>
 <key2>value</key2>
</propertyExpansions>

```

if wish please create issue for it.